CHANGELOG.md

```diff
@@ -12,6 +12,19 @@
 * Correctly format multi-line parentheses in arrow `do` blocks. [Issue
   1144](https://github.com/tweag/ormolu/issues/1144).
 
+* Switched to `ghc-lib-parser-9.12`, with the following new syntactic features:
+   * GHC proposal [#522](https://github.com/ghc-proposals/ghc-proposals/blob/c9401f037cb22d1661931b2ec621925101052997/proposals/0522-or-patterns.rst): `OrPatterns` (enabled by default)
+   * GHC proposal [#569](https://github.com/ghc-proposals/ghc-proposals/blob/c9401f037cb22d1661931b2ec621925101052997/proposals/0569-multiline-strings.rst): `MultilineStrings` (disabled by default)
+   * GHC proposal [#409](https://github.com/ghc-proposals/ghc-proposals/blob/c9401f037cb22d1661931b2ec621925101052997/proposals/0569-multiline-strings.rst): `NamedDefaults` (enabled by default)
```

Comment:
Is the link for `NamedDefaults` incorrect? It is pointing to the multiline strings proposal.

Reply:
Thanks, fixed in 7eb8c79cb26f0cf3657d8f73a36194144cbcf382